### PR TITLE
Bugs v1.1 (#75)

### PR DESCRIPTION
**Pull request by hadarhubara10**

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          (l.port ? ':' + l.port : '') +
+          l.pathname
+            .split('/')
+            .slice(0, 1 + pathSegmentsToKeep)
+            .join('/') +
+          '/?/' +
+          l.pathname.slice(1).split('/').slice(pathSegmentsToKeep).join('/').replace(/&/g, '~and~') +
+          (l.search ? '&' + l.search.slice(1).replace(/&/g, '~and~') : '') +
+          l.hash
+      )
```
Comment (forshtat) [inline public/404.html:23]: Hey @hadarhubara10 
Can you please explain what is going on here?

File: public/index.html [hunk 2]
```diff
@@ -24,6 +24,18 @@
...
+      (function(l) {
+        if (l.search[1] === '/' ) {
+          var decoded = l.search.slice(1).split('&').map(function(s) { 
+            return s.replace(/~and~/g, '&')
+          }).join('?');
+          window.history.replaceState(null, null,
+              l.pathname.slice(0, -1) + decoded + l.hash
+          );
+        }
+      }(window.location))
```
Comment (forshtat) [inline public/index.html:37]: Same question here, actually. Why is this code necessary?

File: src/components/molecules/SuccessModal.tsx [hunk 3]
```diff
@@ -9,14 +9,16 @@ const SuccessModal: FC = () => {
...
+  const [, setFunds] = useLocalStorage<string>('funds', '0.5')
   const [open, setOpen] = useState<boolean>(true)
   const dispatch = useAppDispatch()
 
   useEffect(() => {
     setToken('')
     setHashes({})
     setLocalMintAmount('')
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+    setFunds('0.5')
```
Comment (forshtat) [inline src/components/molecules/SuccessModal.tsx:20]: What is '0.5' ?

File: src/feature/Relay/Commands/RegisterRelay/FundRelay/Funder.tsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ interface IProps {
 export default function Funder({ success }: IProps) {
   const [hashes, setHashes] = useLocalStorage<Hashes>('hashes', {})
   const [listen, setListen] = useState(false)
-  const [funds, setFunds] = useState<number>(0.5)
+  const [funds, setFunds] = useLocalStorage<string>('funds', '0.5')
```
Comment (forshtat) [inline src/feature/Relay/Commands/RegisterRelay/FundRelay/Funder.tsx:18]: General question - I see we store a lot of things in the local storage. When do we clean it up? Will there be an issue if the user wants to register two relays in a row?

File: public/404.html (hunk 1 above)
Comment (hadarhubara10) [reply public/404.html:23]: For the 404 bug, here is the solution from github - https://github.com/anitab-org/stem-diverse-tv-cms/issues/48#issuecomment-1120336354.

File: public/index.html (hunk 2 above)
Comment (hadarhubara10) [reply public/index.html:37]: same for the 404 bug, here is the solution from github - https://github.com/anitab-org/stem-diverse-tv-cms/issues/48#issuecomment-1120336354.

File: src/components/molecules/SuccessModal.tsx (hunk 3 above)
Comment (hadarhubara10) [reply src/components/molecules/SuccessModal.tsx:20]: The recommended amount and default value of the fund
image

File: src/feature/Relay/Commands/RegisterRelay/FundRelay/Funder.tsx (hunk 4 above)
Comment (hadarhubara10) [reply src/feature/Relay/Commands/RegisterRelay/FundRelay/Funder.tsx:18]: This will not be a problem, this is cleaned up in the success modal

**Pull request merged**